ImageFeed/SingleImage/SingleImageViewController.swift

```diff
@@ -0,0 +1,79 @@
+//
+//  SingleImageViewController.swift
+//  ImageFeed
+//
+//  Created by Dmitry Khitrin on 10/10/2022.
+//
+
+import UIKit
+
+class SingleImageViewController: UIViewController {
+	var image: UIImage! {
+		didSet {
+			guard isViewLoaded else { return }
+			singleImage.image = image
```

Comment:
заметьте, что вы верно сделали, выделив две операции на singleImage в одну функцию `setImage`, и здесь выставить новое изображение недостаточно, ведь нужно еще и перескейлить. Давайте исправим  
```suggestion
			setImage()
```